connector/src/main/java/org/geysermc/connector/network/translators/java/world/JavaChunkDataTranslator.java

```diff
@@ -62,7 +62,11 @@ public void translate(ServerChunkDataPacket packet, GeyserSession session) {
                 levelChunkPacket.setChunkX(packet.getColumn().getX());
                 levelChunkPacket.setChunkZ(packet.getColumn().getZ());
                 levelChunkPacket.setData(payload);
-                session.getUpstream().sendPacket(levelChunkPacket);
+                try {
+                    session.getUpstream().sendPacket(levelChunkPacket);
```

Comment:
This is not how the connection being closed should be handled.

Reply:
then how should it